src/librustc_parse/lexer/mod.rs

```diff
@@ -71,6 +70,10 @@ impl<'a> StringReader<'a> {
         }
     }
 
+    pub fn get_pos(&self) -> BytePos {
```

Comment:
```suggestion
    pub fn pos(&self) -> BytePos {
```
https://rust-lang.github.io/api-guidelines/naming.html?highlight=getter#getter-names-follow-rust-convention-c-getter